workflow-runtime/src/test/java/com/squareup/workflow1/RenderWorkflowInTest.kt

```diff
@@ -485,5 +507,85 @@ class RenderWorkflowInTest {
     )
   }
 
+  // https://github.com/square/workflow-kotlin/issues/224
+  @Test fun `exceptions from Snapshots don't fail runtime`() {
```

Comment:
Can you add to these test names 'don't fail runtime' shadows the fact that you are also testing that they *do fail* in Snapshot builder or equals when they should (i.e. don't get swallowed).

Reply:
Fair enough, although I think the better fix here is to not test the thing this test does not actually care about (stop testing that `throw` works).